eessi/testsuite/hooks.py

```diff
@@ -435,7 +434,7 @@ def _set_or_append_valid_systems(test: rfm.RegressionTest, valid_systems: str):
         warn_msg = f"valid_systems has multiple ({len(test.valid_systems)}) items,"
         warn_msg += " which is not supported by this hook."
         warn_msg += " Make sure to handle filtering yourself."
-        warnings.warn(warn_msg)
```

Comment:
Just to double check: these changes were _not_ accidental, right?

Reply:
I don't mind too much, but you _could_ set `bench_name=None` in the class body. Then, with the current logic of setting `self.banch_name=self.bench_name_ci` in that one particular case in the init hook, that should work, right? I.e. it's essentially only one extra line in the test (a default bench_name).

Reply:
you're right, i readded the bench_name check in the mixin class.
i now also set `bench_name_ci` only for the CI test to avoid that `bench_name` always has to be set to something.

Reply:
and yes, those warning changes are again *not* accidental :)